{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:python:python:3.8.11:*:*:*:*:*:*:*",
	"cpeNameId": "4B3183F9-BDAC-45A7-8B88-2E9DC922DADB",
	"created": "2022-02-14T18:13:34.117",
	"lastModified": "2022-05-20T13:25:07.680",
	"titles": [
		{
			"title": "Python 3.8.11",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://bugs.python.org/issue43882",
			"type": "Advisory"
		},
		{
			"ref": "https://github.com/python/cpython/tags",
			"type": "Change Log"
		}
	]
}
